### DISCUSSION TITLE
Serenity Login Page add textbox

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5348] [Help]
I want to add texbox to serenity login page
I want it to be connected to different databases according to this code entered later.
how can i do this
For example like this:
![image](https://user-images.githubusercontent.com/24252615/104419238-3bf93380-5589-11eb-88fe-77b6be5adad1.png)

**Comment by VictorTomaili** [comment]
Add a new connection string to your configuration. Then create row related to this connection string. 
Add your new field to [LoginRequest](https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/Membership/Account/LoginRequest.cs) with `ClientSide` attribute. Modify your login request in [LoginPanel](https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/Membership/Account/LoginPanel.ts) and [AccountController](https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/Membership/Account/AccountPage.cs#L53)

**Comment by edwardch** [comment]
To connect to different databases take a look at the wiki ([Database: Multi Database Serenity](https://github.com/serenity-is/Serenity/wiki/Database:-Multi-Database-Serenity)) for inspiration

**Reply by havvanur160401046** [reply]
yes this is exactly what I want to do
![image](https://user-images.githubusercontent.com/24252615/104566459-aaf48c00-565e-11eb-99a6-f688e37fd600.png)
Under which folder will this class be created?

**Reply by edwardch** [reply]
Probably somewhere in Administration like User/Auth

**Reply by havvanur160401046** [reply]
![image](https://user-images.githubusercontent.com/24252615/104592744-ba380180-567f-11eb-8fcc-d89af406d926.png)
![image](https://user-images.githubusercontent.com/24252615/104592832-df2c7480-567f-11eb-879b-19a98bdf91eb.png)
I get an error like this. LookupEditor what should I write

**Reply by havvanur160401046** [reply]
@edwardch

**Comment by havvanur160401046** [comment]
![image](https://user-images.githubusercontent.com/24252615/104575651-50f8c400-5668-11eb-8e52-6334e9c5cc8e.png)
I get an error like this

**Reply by VictorTomaili** [reply]
![image](https://user-images.githubusercontent.com/2023440/104585116-a3d87880-5674-11eb-8f5a-2cd1f9ae9f56.png)
